src/fmu/dataio/dataio.py

```diff
@@ -236,15 +236,15 @@ def __init__(
             "Inside RMS status (developer setting) is %s",
             kwargs.get("inside_rms", False),
         )
-
         # When running RMS, we are in conventionally in RUNPATH/rms/model, while
         # in other settings, we run right at RUNPATH level (e.g. ERT jobs)
         if self._runpath and isinstance(self._runpath, (str, pathlib.Path)):
             self._runpath = pathlib.Path(self._runpath).absolute()
             logger.info("The runpath is hard set as %s", self._runpath)
-        elif self._runpath is None and ("RMS_COMPANYIMAGESPATH" in os.environ):
+        elif self._runpath is None and ("RMS_ENABLE_HAVANA_EXPORT" in os.environ):
```

Comment:
`if "rms" in sys.executable and "komodo" not in sys.executable` is maybe an option? the "komodo not in" is a safeguard against a komodo-release including the three letters "rms" which could happen. Still, it might depend on our Equinor-deploy of RMS.

Reply:
Yes it is possibly a more robust solution. Will give it a go.